tools/config-generator/main.go

```diff
@@ -612,8 +622,7 @@ func parseBasicJobConfigOverrides(data *baseProwJobTemplateData, config yaml.Map
 		(*data).ExtraRefs = append((*data).ExtraRefs, "  "+(*data).PathAlias)
 	}
 	if needGo114 {
-		data.addEnvToJob("GO_VERSION", "go1.14")
-		(*data).Image = getGo114ImageName((*data).Image)
+		data.SetGoVersion("go1.14")
```

Comment:
Probably better to define `go1.14` as a constant, as it's used multiple times.

Reply:
also done, more-or-less